ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from client.models import Client
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from ordered_product.models import OrderedProduct
+from django.contrib.auth.models import User
+
+
+USER_NAME = 'meitar'
+PASSWORD = 'pass1234'
+VALID_PRODUCT_NAME = 'Banana'
+INVALID_PRODUCT_NAME = 'Eggplant'
+USER_NAME = 'meitar'
+VALID_SUPPLIER_PRODUCT_ID = '1'
+INVALID_SUPPLIER_PRODUCT_ID = '0'
+VALID_DELIVERY_LOCATION_ID = '3'
+INVALID_DELIVERY_LOCATION_ID = '9797'
+VALID_QUANTITY = '5'
+ORDER_PRODUCT_ID = '1'
+
+
+class Test_order_list:
+    @pytest.mark.django_db()
+    def test_empty_orders_list(self, client):
+        user = User.objects.create(username='testuser')
+        user.set_password('12345')
+        user.save()
+        client.login(username='testuser', password='12345')
+        response = client.get('/ordered_product/orders')
+        assert response.status_code == 200
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert 'ordered_product/orders.html' in template_names
+        assert set(response.context['orders']) == set(OrderedProduct.filter_user_name(Client(user)))
```

Comment:
It will be way more readable if you'll separate this to 3 lines:
one defining the variable on the right
one defining the variable on the left
and last one for the assertions.

make sure to name the variables well, so it'll be clear what you're testing, even if the reviewer does not know Django or HTML